rakelib/vic.rake

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require './rakelib/support/vic_load_test'
+
+namespace :vic do
+  N = 10
```

Comment:
@patrickvinograd Wasn't sure how large we wanted this, so it's at 10 for now to make development of the scripts easier and we can bump it up during testing.